packages/webapp/src/components/Form/MultiStepForm/WithStepperProgressBar.tsx

```diff
@@ -126,6 +140,13 @@ export const WithStepperProgressBar = ({
           />
         </FloatingContainer>
       )}
+      {transition.unblock && (
```

Comment:
Just wondering if we should add a check for if the form is dirty? Then we could skip the exit modal..

I am mostly asking because I leave the form without changing anything as a developer a fair amount -- but the user might like it too haha